CMakeLists.txt

```diff
@@ -874,7 +874,7 @@ cmake_dependent_option(
   "Whether to build the flash_attention kernel for scaled dot product attention.\
   Will be disabled if not supported by the platform"
   ON
-  "USE_CUDA OR USE_ROCM;NOT MSVC"
+  "USE_CUDA OR USE_ROCM"
```

Comment:
This code if executed on MSVC CUDA will use USE_FLASH_ATTENTION = ON. I believe this whats most likely caused OOM

Reply:
Indeed this could have broken it. Let me fix it right away!